src/main/java/ITNews/project/ITNews/service/CloudinaryService.java

```diff
@@ -0,0 +1,84 @@
+package ITNews.project.ITNews.service;
+
+import ITNews.project.ITNews.dto.ControllerResponse;
+import ITNews.project.ITNews.model.ImgNewsEntity;
+import ITNews.project.ITNews.repository.ImgNewsRepository;
+import com.cloudinary.Cloudinary;
+import com.cloudinary.utils.ObjectUtils;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@Service
+@Transactional
+public class CloudinaryService {
+    private static final String noImg= "https://res.cloudinary.com/itnewscloud/image/upload/v1597912961/No-image-available_dljkwb.png";
```

Comment:
hardcoded value it's bad practice

Reply:
refactoring

    @Value("${urlCloud}")
    private String noImg;